src/scripts/modules/media/body/content-baked.less

```diff
@@ -240,7 +240,17 @@ h1.example-title .text {
 
 // Tables
 // --------------------------------------------------
+.swipe-table {
```

Comment:
Maybe I'm misunderstanding but if the JS _just_ added a `.is-not-mobile` class on the root element then couldn't we just add something like the following:

```css
.is-not-mobile {
  overflow-x: scroll; /* or : auto; , my CSS is a bit rusty */
}
```

Then, there would be no need to make sure JS listeners were added whenever the page content changed, right?

Reply:
You have to add listeners to enable users to swipe tabels. It's all about allowing users to swipe instead of using normal scrollbar

Reply:
ok, thanks for the explanation. Could we still use the event delegation rather than attaching these listeners to the DOM nodes directly? That would help with the issue described in the Pull Request description (needing to re-add the listener when the page changes)